.devcontainer/Dockerfile

```
@@ -1,26 +1,29 @@
-ARG VARIANT="8.2-apache-bullseye"
-FROM mcr.microsoft.com/vscode/devcontainers/php:${VARIANT}
+FROM php:8.4-apache
 
-ARG DEBIAN_FRONTEND=noninteractive
-ARG apcu_version=5.1.23
-ARG memcached_version=3.2.0
-ARG redis_version=6.0.2
-ARG imagick_version=3.7.0
+ARG DEBIAN_FRONTEND=noninteractive´
```

Not sure how this happened...

```suggestion
ARG DEBIAN_FRONTEND=noninteractive
```